source/cards/deck.py

```diff
@@ -32,6 +32,9 @@ def shuffle(self):
     def get_random_card(self):
         return self.cards.pop()
 
```

Comment:
This function should be deleted since we have the 'get_random_cards' function which returns 1 card as default